src/function/getJournal.ts

```diff
@@ -0,0 +1,28 @@
+import { Context, Callback } from 'aws-lambda';
+
+import createResponse from '../util/createResponse';
+import journal from '../service/journal';
+
+export function getJournal(event: any, context: Context, callback: Callback) {
+
+  const journalService = new journal();
```

Comment:
Please make J capital in **new journal**

Reply:
TSLint says it should be lowercase

Reply:
I think you need to import it as Journal